packages/typespec-ts/static/static-helpers/serialization/serialize-record.ts

```diff
@@ -0,0 +1,9 @@
+export function serializeRecord(item: any, excludeProperties: string[] = []) {
```

Comment:
Can we take value type ser/deserialization function as a parameter for it? instead of use `as any`?

Reply:
Currently all additional properties share the same helper, do you mean we build one helper for every additional property?